config/debug_traject.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
```

Comment:
@awead this file can actually be ignored, I just added this to be able to debug easier.